docs/install/source.md

```diff
@@ -0,0 +1,121 @@
+# Install from Source
+
+Install OFRAK from source code for development or contribution.
+
+## Prerequisites
+
+- Python 3.9+ and pip
+- Git with [Git LFS](https://git-lfs.github.com/) installed ([installation instructions](https://github.com/git-lfs/git-lfs#installing))
+- make
+
+## Clone and Install
+
+```bash
+# Clone repository
+git clone https://github.com/redballoonsecurity/ofrak.git
+cd ofrak
+
+# Install Git LFS files
+git lfs install
+git lfs pull
+```
+
+## Install Packages
+
+### Easy Way: One Command
+
```

Comment:
I know it's not exactly "one command," but this feels like the right place to recommend using a virtual environment...

Reply:
Added tip in here.